build.rs

```diff
@@ -212,7 +212,12 @@ fn which_freebsd() -> Option<i32> {
 }
 
 fn emcc_version_code() -> Option<u64> {
+    #[cfg(target_os = "windows")]
+    let output = Command::new("emcc.bat").arg("-dumpversion").output().ok()?;
+
+    #[cfg(not(target_os = "windows"))]
```

Comment:
Nit: could you use `if cfg!(target_os = "windows") { ... }` here instead? `cfg!(...)` is slightly preferred over `#[cfg(...)]` where possible since it allows the code to be checked on all OSs.

Otherwise lgtm, fyi @hoodmane.